structuredheadings/plugin.js

```diff
@@ -142,6 +184,16 @@
           command: "matchHeading",
           toolbar: "styles,1"
         });
+        editor.ui.addButton("increaseheadinglevel", {
+          label: "Increase Heading Level",
+          command: "increaseHeadingLevel",
+          toolbar: "styles,2"
+        });
+        editor.ui.addButton("decreaseheadinglevel", {
```

Comment:
idea:  is it possible to unify the capitalization between command & button/icon name and then just use that constant?